contributors/design-proposals/dir_struct.txt

```diff
@@ -0,0 +1,240 @@
+Uncategorized (Please Help)
+	high-availability.md
+	control-plane-resilience.md
+	downward_api_resources_limits_requests.md
+	seccomp.md
+	client-package-structure.md
+	service-discovery.md
+	metadata-policy.md
+	containerized-mounter.md~
+	identifiers.md
+	local-cluster-ux.md
+	pod-pid-namespace.md
+	grow-volume-size.md
+	image-provenance.md
+	core-metrics-pipeline.md
+	versioning.md
+	ha_master.md
+	secret-configmap-downwarapi-file-mode.md
+	protobuf.md
+	flakiness-sla.md
+	resources.md
+	initial-resources.md
+	create_sheet.py
+	runtime-client-server.md
+	OWNERS
+	namespaces.md
+	cpu-manager.md
+	selinux-enhancements.md
+	sysctl.md
+	dir_struct.txt
+	selinux.md
+	templates.md
+	pod-cache.png
+	README.md
+	multi-platform.md
+	pod-lifecycle-event-generator.md
+	secrets.md
+	cri-dockershim-checkpoint.md
+	event_compression.md
+	pleg.png
+./sig-cli
+	kubectl-extension.md
+	kubectl_apply_getsetdiff_last_applied_config.md
+	multi-fields-merge-key.md
+	template.md
```

Comment:
Is this different from templates.md?

Reply:
My mistake.  This is the design proposal template